### DISCUSSION TITLE
Fix plaintext rendering of text in notification emails

### DISCUSSION THREAD

**Discussion by ckreibich** [community/community#16931] [Pull Requests]
Some characters that look fine on github.com end up URL-encoded in plaintext email. For example,

    check out this "quoted" string

ends up in the notification email as

    check out this &quot;quoted&quot; string

which quickly becomes painful to parse. It'd be nice to see this unescaped.

**Comment by ThiefMaster** [comment]
Has it really been 4 years since text/plain notifications became shitty?! I think lately they got much worse...
